packages/itwinui-css/src/tile/tile.scss

```diff
@@ -353,19 +352,24 @@ $thumbnail-selector: ':is(.iui-thumbnail-icon, .iui-tile-thumbnail-picture)';
     justify-self: end;
   }
 
-  .iui-tile-thumbnail-picture ~ & {
-    @include mixins.iui-blur($opacity: 5);
-    --_iui-button-icon-fill: var(--iui-color-white);
+  .iui-button[data-iui-variant='borderless'] {
+    border-radius: var(--iui-border-radius-round);
 
-    > .iui-button-icon {
-      filter: drop-shadow(0 2px 1px rgba(0, 0, 0, var(--iui-opacity-5)));
-    }
+    .iui-tile-thumbnail-picture ~ & {
+      @include mixins.iui-blur($opacity: 5);
+      --_iui-field-color-icon: var(--iui-color-white);
+      --_iui-field-color-icon-hover: var(--iui-color-white);
+
+      > .iui-button-icon {
+        filter: drop-shadow(0 2px 1px rgba(0, 0, 0, var(--iui-opacity-5)));
+      }
 
-    &:hover,
-    &[data-iui-active='true']:enabled,
-    &[data-iui-active='true']:enabled:hover,
-    &[data-iui-active='true']:active {
-      background-color: rgba(0, 0, 0, var(--iui-opacity-4));
+      &:hover,
+      &[data-iui-active='true']:enabled,
+      &[data-iui-active='true']:enabled:hover,
+      &[data-iui-active='true']:active {
+        background-color: rgba(0, 0, 0, var(--iui-opacity-4));
+      }
```

Comment:
Tile was failing React tests.  Looked into it and our CSS demo's HTML structure was not matching React's.

**React:**
```html
<div class='iui-tile-thumbnail-quick-action'>
  <button class="iui-button iui-button-base iui-field">…</button>
</div>
```

**CSS:**
```html
<button class="iui-button iui-button-base iui-field iui-tile-thumbnail-quick-action">…</button>
```

Made the changes in CSS demos to match React & had to update the CSS to work properly.  The hover state of the buttons overlaying the thumbnail aren't even working properly in prod right now: https://itwin.github.io/iTwinUI/react/?mode=preview&story=tile--all-props

Reply:
I added this to the changesets.  Unsure if this warrants a major or not.